chemie/elections/views.py

```diff
@@ -375,3 +390,64 @@ def admin_end_election(request):
             )
     election.end_election()
     return redirect('elections:results')
+
+
+@permission_required('elections.add_election')
+@login_required
+def change_rfid_status(request):
+    if request.method == "POST":
+        form = GetRFIDForm(request.POST)
+        if form.is_valid():
+            rfid = form.cleaned_data.get('rfid')
+            em_code = ProfileManager.rfid_to_em(rfid)
+            try:
+                profile = Profile.objects.get(access_card=em_code)
+            except:
+                messages.add_message(request, messages.WARNING, 'Studentkortnummeret er ikke registrert enda.')
+                return redirect('elections:add_rfid')
+            eligible = profile.eligible_for_voting
```

Comment:
Heile blokka kan erstattes med:
`profile.eligible_for_voting = not profile.eligible_for_voting`